make/autoconf/lib-hsdis.m4

```diff
@@ -134,17 +134,23 @@ AC_DEFUN([LIB_BUILD_BINUTILS],
   BINUTILS_SRC="$with_binutils_src"
   UTIL_FIXUP_PATH(BINUTILS_SRC)
 
+  BINUTILS_DIR="${OUTPUTDIR}/binutils"
```

Comment:
```suggestion
  BINUTILS_DIR="$CONFIGURESUPPORT_OUTPUTDIR/binutils"
```

Rationale: "configure-support" includes things that are created by the configure script. We try to avoid cluttering the top level build directory.